test/python/WMCore_t/MicroService_t/MSPileup_t/MSPileupMonitoring_t.py

```diff
@@ -0,0 +1,60 @@
+"""
+Unit tests for MSPileupMonitoring.py module
+
+Author: Valentin Kuznetsov <vkuznet [AT] gmail [DOT] com>
+"""
+
+# system modules
+import unittest
+
+# WMCore modules
+from WMCore.MicroService.MSPileup.MSPileupMonitoring import flatDocuments, flatKey
+
+
+class MSPileupMonitoringTest(unittest.TestCase):
+    "Unit test for MSPileupMonitoring module"
+
+    def setUp(self):
+        rses = ['rse1', 'rse2']
+        campaigns = ['c1', 'c2']
+        ruleIds = ['1', '2']
+        self.doc = {
+            'pileupName': '/klsjdfklsd/klsjdflksdj/PREMIX',
+            'pileupType': 'classic',
+            'expectedRSEs': rses,
+            'currentRSEs': rses,
+            'fullReplicas': 1,
+            'campaigns': campaigns,
+            'containerFraction': 0.0,
+            'replicationGrouping': "ALL",
+            'active': True,
+            'pileupSize': 0,
+            'ruleIds': ruleIds}
+
+    def testMSPileupMonitoring(self):
```

Comment:
This is not really testing MSPileupMonitoring class, but the utilitarian functions.
As such, I would suggest two minor changes here:
1) give a more meaningful test name, e.g.: `testFlatKeys`
2) whenever possible, create unit tests that are as small as possible. In this case, I would suggest to actually have 2 unit tests, one for each function.

Reply:
done